staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_controller.go

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
if the dependencies allow it, moving this and possibly the aggregator to pkg/controllers/openapi would be great.

Reply:
Done in next commit.